ducksteroid/Test/RankingTests.cs

```diff
@@ -0,0 +1,128 @@
+using NUnit.Framework;
+using System.Collections.Generic;
+using System;
+using MyGame.src;
+
+namespace Tests
+{
+    public class RankingTests
+    {
+        //set-up method to return a ranking obj with 4 scores in it
+        public Ranking Setup()
+        {
+            Ranking rank = new Ranking();
+            Score score1 = new Score(20);
+            Score score2 = new Score(15);
+            Score score3 = new Score(2);
+            Score score4 = new Score(15);
+            rank.AddScore(score1);
+            rank.AddScore(score2);
+            rank.AddScore(score3);
+            rank.AddScore(score4);
+            return rank;
+        }
+
+        //test: check if ranking adds the correct score
+        //expected: contain the correct score reference (true)
+        [Test]
+        public void TestAddCorrectScore()
+        {
+            Ranking rank = Setup();
+            Score score1 = new Score(20);
+            rank.AddScore(score1);
+            bool actual = rank.Scores.Contains(score1);
+
+            Assert.AreEqual(true, actual, "Test ranking can add score");
+        }
+        //test: check if file is saved correctly
+        //expected: number of scores in the ranking obj
+        [Test]
+        public void TestSaveFile1()
+        {
+            Ranking rank = Setup();
+            rank.SaveScores("scores.txt");
```

Comment:
The idea for refactoring is we should have 1 more 'test' folder in the Resource folder to store the test result. Otherwise, it will affect the existing score.